### PR TITLE
MWPW-121861 - Text block updates to support SEO Long form content

### PR DESCRIPTION
**Pull request by ryanmparrish**
* Updated the `Text` block to support `Inset` and `Long Form` variants
* Added a block typography config for variant mapping and optional overrides
* Extended the block content container style options to target token widths per viewport

Resolves: [MWPW-121861](https://jira.corp.adobe.com/browse/MWPW-121861)

**Test URLs:**
- [Before](https://main--milo--adobecom.hlx.page/drafts/rparrish/text-block-variants?martech=off)
- [After](https://rparrish-con-blocks--milo--adobecom.hlx.page/drafts/rparrish/text-block-variants?martech=off)

### CODE REVIEW

**Review by Sartxi — APPROVED** [review]
Looks good to me 👍🏻

File: libs/blocks/text/text.js [hunk 1]
```diff
@@ -13,17 +26,31 @@ export default function init(el) {
...
     const [head, ...tail] = rows;
     decorateBlockBg(el, head);
     rows = tail;
   }
+  const helperClasses = [];
+  let blockType = 'text';
   const size = getBlockSize(el);
-  decorateBlockText(el, blockTypeSizes.text[size]);
+  const longFormVariants = ['inset', 'long-form', 'bio'];
+  longFormVariants.forEach((v, i) => {
```
Comment (rgclayton) [inline libs/blocks/text/text.js:38]: Nit: one letter parameters are a little difficult to understand.

File: libs/blocks/text/text.js [hunk 2]
```diff
@@ -13,17 +26,31 @@ export default function init(el) {
...
+    if (el.classList.contains(v)) {
+      helperClasses.push('max-width-8-desktop');
+      blockType = (i > 0) ? 'standard' : v;
+    }
+  });
+  const config = blockTypeSizes[blockType][size];
+  const overrides = ['-heading', '-body', '-detail'];
+  overrides.forEach((o, i) => {
```
Comment (rgclayton) [inline libs/blocks/text/text.js:46]: Sane nit.

File: libs/blocks/text/text.js [hunk 3]
```diff
@@ -13,17 +26,31 @@ export default function init(el) {
...
+    const hasClass = [...el.classList].filter((c) => c.includes(o));
+    if (hasClass.length) config[i] = hasClass[0].split('-').shift().toUpperCase();
```
Comment (rgclayton) [inline libs/blocks/text/text.js:48]: Same nit

File: libs/blocks/text/text.js (hunk 1 above)
Comment (ryanmparrish) [reply libs/blocks/text/text.js]: Agree, I updated this to use better function var names.

File: libs/blocks/text/text.js (hunk 2 above)
Comment (ryanmparrish) [reply libs/blocks/text/text.js]: Same 👍

**Review by elan-tbx — APPROVED** [review]
LGTM

**Comment by ryanmparrish** [comment]
Long form content lets go!

**Review by rgclayton — COMMENTED** [review]
👍

**Pull request merged**